lib/mail/version_specific/ruby_1_9.rb

```diff
@@ -89,7 +89,7 @@ def Ruby19.b_value_decode(str)
       decoded.valid_encoding? ? decoded : decoded.encode(Encoding::UTF_16LE, :invalid => :replace, :replace => "").encode(Encoding::UTF_8)
     rescue Encoding::UndefinedConversionError, ArgumentError, Encoding::ConverterNotFoundError
       warn "Encoding conversion failed #{$!}"
-      str.dup.force_encoding(Encoding::UTF_8)
+      str.dup
```

Comment:
So without the encoding forced, this will allow exceptions to be thrown elsewhere as needed, right?